compiler/ir/ir.psi2ir/src/org/jetbrains/kotlin/psi2ir/generators/FunctionGenerator.kt

```diff
@@ -71,19 +73,20 @@ class FunctionGenerator(declarationGenerator: DeclarationGenerator) : Declaratio
             ?.let {
                 declareSimpleFunctionInner(it, ktElement, IrDeclarationOrigin.FAKE_OVERRIDE)
                     .buildWithScope { irFunction ->
-                        generateFunctionParameterDeclarationsAndReturnType(irFunction, ktElement, null)
+                        generateFunctionParameterDeclarationsAndReturnType(irFunction, ktElement, null, emptyList())
```

Comment:
Need a test with a lambda used where a function type with context receivers is expected (`context (A) B.() -> C` or something).

Reply:
Added a simple one in [this commit](https://github.com/JetBrains/kotlin/pull/4488/commits/cdf1e5283ebf50b2278c79bec7ebf397e5d02d21).